docs/Makefile

```diff
@@ -79,7 +79,7 @@ production-build: clean generate hugo-production-build
 	make live-blocks-inject
 
 .PHONY: preview-build
-preview-build:
+preview-build: dev-generate
```

Comment:
This is called through the top-level goal, and that already includes the `dev-generate`, [see here](https://sourcegraph.com/github.com/open-policy-agent/opa/-/blob/Makefile?L450). I'm sure it could be optimized for local usage etc, though.